### PR TITLE
Added SendMessagesInThreads and StartEmbeddedActivities.

### PR DESCRIPTION
**Pull request by drobbins329**
Adjusted owner perms.
Change UsePublicThreads -> CreatePublicThreads
Change UsePrivateThreads -> CreatePrivateThreads

### CODE REVIEW

File: src/Discord.Net.Core/Entities/Permissions/GuildPermission.cs [hunk 1]
```diff
@@ -191,18 +191,26 @@ public enum GuildPermission : ulong
         ///     authentication when used on a guild that has server-wide 2FA enabled.
         /// </remarks>
         ManageThreads           = 0x04_00_00_00_00,
-        /// <summary>
-        /// Allows for creating and participating in threads.
+        /// /// <summary>
```
Comment (MrCakeSlayer) [inline src/Discord.Net.Core/Entities/Permissions/GuildPermission.cs]: Remove the extra slashes please :)

Comment (drobbins329) [reply src/Discord.Net.Core/Entities/Permissions/GuildPermission.cs]: oops

**Review by MrCakeSlayer — APPROVED** [review]

**Comment by drobbins329** [comment]
removed extra ///

File: src/Discord.Net.Core/Entities/Permissions/GuildPermission.cs [hunk 2]
```diff
@@ -192,17 +192,25 @@ public enum GuildPermission : ulong
         /// </remarks>
         ManageThreads           = 0x04_00_00_00_00,
         /// <summary>
-        /// Allows for creating and participating in threads.
+        ///     Allows for creating public threads.
         /// </summary>
-        UsePublicThreads        = 0x08_00_00_00_00,
```
Comment (quinchs) [inline src/Discord.Net.Core/Entities/Permissions/GuildPermission.cs:197]: Could you add back `UsePublicThreads` and `UsePrivateThreads` with the `[Obsolete]` attribute explaining the rename?

**Review by quinchs — CHANGES_REQUESTED** [review]

File: src/Discord.Net.Core/Entities/Permissions/GuildPermission.cs (hunk 2 above)
Comment (drobbins329) [reply src/Discord.Net.Core/Entities/Permissions/GuildPermission.cs:197]: Added back to GuildPermission.cs, with [Obsolete], directing to Create...Threads and SendMessagesInThreads.

**Review by quinchs — APPROVED** [review]

**Pull request merged**